rtos/ConditionVariable.h

```diff
@@ -167,8 +167,8 @@ class ConditionVariable : private mbed::NonCopyable<ConditionVariable> {
      * @note - Spurious notifications can occur, so the caller of this API
      * should check to make sure the condition the caller is waiting on has
      * been met.
-     * 
-     * @note - The current thread will release the lock while inside the wait
+     *
+     * @note - The current thread releases the lock while inside the wait
```

Comment:
..."and reacquires it" below

Reply:
Dunno why this comemnt wasn't auto-dismissed. Looks like it was changed.